# HG changeset patch
# User jwe
# Date 745390167 0
#      Sun Aug 15 04:49:27 1993 +0000
# Node ID 50af4412a25f2fac346e34c3c5db7a43293d2a7c
# Parent  671f8bf989d8dcda13ebdc9147b1415e05b91869
[project @ 1993-08-15 04:49:27 by jwe]
(text_functions): Add run_history to the list.

diff --git a/src/builtins.cc b/src/builtins.cc
--- a/src/builtins.cc
+++ b/src/builtins.cc
@@ -175,16 +175,19 @@ static builtin_text_functions text_funct
     "print/save/load command history\n", },
 
   { "load", -1, builtin_load,
     "load variables from a file\n", },
 
   { "ls", -1, builtin_ls,
     "print a directory listing\n", },
 
+  { "run_history", -1, builtin_run_history,
+    "usage: run_history [first] [last]\n", },
+
   { "save", -1, builtin_save,
     "save variables to a file\n", },
 
   { "set", -1, builtin_set,
     "set plotting options\n", },
 
   { "show", -1, builtin_show,
     "show plotting options\n", },
